main.js

```diff
@@ -0,0 +1,104 @@
+var width = 960,
+    height = 500,
+    centered;
+
+// Define color scale
+var color = d3.scale.linear()
+  .domain([1, 20])
+  .clamp(true)
+  .range(['#fff', '#409A99']);
+
+var projection = d3.geo.mercator()
+  .scale(35500)
+  // Center the Map in Colombia
+  .center([-71, 42.4])
+  .translate([width / 2, height / 2]);
+
+var path = d3.geo.path()
+  .projection(projection);
+
+// Set svg width & height
+var svg = d3.select('svg')
+  .attr('width', width)
+  .attr('height', height);
+
+// Add background
+svg.append('rect')
+  .attr('class', 'background')
+  .attr('width', width)
+  .attr('height', height)
+
+var g = svg.append('g');
+
+var effectLayer = g.append('g')
+  .classed('effect-layer', true);
+
+var mapLayer = g.append('g')
+  .classed('map-layer', true);
+
+var dummyText = g.append('text')
+  .classed('dummy-text', true)
+  .attr('x', 10)
+  .attr('y', 30)
+  .style('opacity', 0);
+
+var bigText = g.append('text')
+  .classed('big-text', true)
+  .attr('x', 20)
+  .attr('y', 45);
+
+// Load map data
+d3.json('geodata.geojson', function(error, mapData) {
+  d3.json('boston.json', function(error, data){
+
+  var features = mapData.features;
+  var points = data.features
+
+  // Update color scale domain based on data
+
+  // Draw each province as a path
+  mapLayer.selectAll('path')
+      .data(points)
+      .enter().append('path')
+      .attr('d', path)
+      .attr('vector-effect', 'non-scaling-stroke')
+      .style('fill', "#A8DBA8")
+  mapLayer.selectAll('path')
+      .data(features)
+      .enter().append('path')
+      .attr('d', path)
+      .attr('vector-effect', 'non-scaling-stroke')
+      .style('fill', "#0B486B")
+  });
+})
+
+var BASE_FONT = "'Helvetica Neue', Helvetica, Arial, sans-serif";
+
+var FONTS = [
```

Comment:
Ditto above: 23 unused fonts is a LOT of unneeded extra load time (probably at least a couple full seconds). Don't see them used, so we should probably drop them out.